.wordlist.txt

```diff
@@ -42,13 +43,14 @@ GPGPU
 GWS
 hardcoded
 HC
+hcBLAS
 HIP's
 hipcc
 hipexamine
 hipified
 hipother
 HIPRTC
-hcBLAS
+hypothetic
```

Comment:
```suggestion
```

I changed this to hypothetical below (which is a more standard usage), so you can remove this.

Reply:
Thanks! Updated.